include/boost/json/value.hpp

```diff
@@ -2215,7 +2215,7 @@ class value
 
     /** Return the underlying `std::uint64_t`, or throw an exception.
 
-        If @ref is_int64() is `true`, returns
+        If @ref is_uint64() is `true`, returns
```

Comment:
`is_uint64` returns a pointer not a bool, same with the others

Reply:
How should it be worded then? "If is_uint64_t() returns a non-null pointer"?

Reply:
actually we are going to roll back the pointer change, stay tuned